AppKit/CPTrackingArea.j

```diff
@@ -0,0 +1,141 @@
+/*
+ * CPTrackingArea.j
+ * AppKit
+ *
+ * Created by Didier Korthoudt.
+ * Copyright 2015, Cappuccino Project.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+@import <Foundation/Foundation.j>
+@import "CPView.j"
+
+@typedef CPTrackingAreaOptions
+CPTrackingMouseEnteredAndExited     = 1 << 1,
+CPTrackingMouseMoved                = 1 << 2,
+CPTrackingCursorUpdate              = 1 << 3,
+CPTrackingActiveWhenFirstResponder  = 1 << 4,
+CPTrackingActiveInKeyWindow         = 1 << 5,
+CPTrackingActiveInActiveApp         = 1 << 6, // both mean the same in cappuccino context
```

Comment:
Indeed, I could be more complete... What I wan't to explain is that, for platform windows, as browsers won't pass events to an inactive window, specifying CPTrackingActiveInActiveApp or CPTrackingActiveAlways does nothing more than CPTrackingActiveInKeyWindow, and for normal windows, CPTrackingActiveInActiveApp and CPTrackingActiveAlways give the same effect.